extensions/sql-migration/scripts/SHIR-auto-configuration.ps1

```diff
@@ -0,0 +1,620 @@
+﻿###############################################################################################
+# $Description: This script automatically downloads, installs and configures SHIR to DMS on a Windows machine.
+# $Id: SHIR-auto-configuration.ps1
+# $Author: anjaligoyal $ gasachdeva $
+###############################################################################################
+
+<#
+.SYNOPSIS
+Automatically configures SHIR on a Windows machine including downloading, installing Integration runtime,
+and configuring IR with DMS.
+
+.DESCRIPTION
+The script downloads, installs and configures SHIR on a Windows machine.
+
+.PARAMETER AdminPriv
+(Optional) [switch] True if script is running with admin privileges else false.
+
+.PARAMETER AuthKey1
+(Optional) [string] First authentication key to register node to DMS.
+
+.PARAMETER AuthKey2
+(Optional) [string] Second authentication key to register node to DMS.
+#>
+
+param (
+	[switch] $AdminPriv = $false,
+	[string] $AuthKey1 = $null,
+	[string] $AuthKey2 = $null
+)
+
+$timeStamp = [System.DateTime]::Now.ToString("yyyyMMddHHmmss")
+# Unique script id for every run - Telemetry purpose
+$Global:ScriptId = "Script-$timestamp"
+
+# TODO: Auto populate
+$Global:LatestIRVersion = [Version]"5.34.8675.1"
```

Comment:
please double check, I think the latest required version is 5.37

Reply:
[https://go.microsoft.com/fwlink/?linkid=839822] - this is the link we use, which points to 5.34 version